### DISCUSSION TITLE
QueryClient on v10

### DISCUSSION THREAD

**Discussion by luixo** [trpc/trpc#2639] [Q&A]
Hello,
I noticed v10 lost `trpc.useContext().queryClient`. Is there a reason behing that?
I used to pass around `trpcContext` to manipulate cache. Regular `getData()` and similar are not enough for me - I do a broader manipulation that requires `queryClient.getQueriesData`.

**Comment by KATT** [comment]
It's supposed to be there, please raise an issue if you see it on the latest alpha.

FWIW, you can also use it straight from react-query
